doc/content/components/network-server.md

```diff
@@ -7,3 +7,51 @@ weight: 3
 The Network Server handles the LoRaWAN network layer, including MAC commands, regional parameters and adaptive data rate (ADR).
 
 <!--more-->
+
+## Device Management
+
+Network Servers expose `Get`, `Set` and `Delete` gRPC methods for device management. REST analogues for these methods exist as well. Typical clients of these RPC methods are [Console]({{< ref "/components/console.md" >}}) and [CLI]({{< ref "/components/cli.md" >}}).
+
+In case the device has an active session and it is in Class B or C, a downlink task may added to internal downlink task queue following the device update.
```

Comment:
I don't know if we should go into implementation details here. Also we're using the term "downlink task queue" that we haven't explained yet.

Reply:
I think it's safe to say here that if you update device settings that may trigger a downlink message (like desired MAC parameters change), the NS schedules the downlink message.